internal/scorecard/storage.go

```diff
@@ -177,6 +177,13 @@ func addStorageToPod(podDef *v1.Pod, mountPath string, storageImage string) {
 	}
 	podDef.Spec.Containers[0].VolumeMounts = append(podDef.Spec.Containers[0].VolumeMounts, vMount)
 
+        // add mountPath to Env
+        mountPathEnv := v1.EnvVar{
```

Comment:
this fails `go fmt`. Should be a TAB.
```suggestion
     mountPathEnv := v1.EnvVar{
```